IdentityCore/src/cache/key/MSIDDefaultAccountCacheQuery.m

```diff
@@ -35,16 +35,6 @@ - (NSString *)account
     return nil;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->